app/models/miq_request_task/state_machine.rb

```diff
@@ -3,7 +3,7 @@ module MiqRequestTask::StateMachine
   delegate :my_zone, :to => :source,      :allow_nil => true
 
   def my_task_id
-    "#{self.class.base_model.name.underscore}_#{id}"
+    "r#{miq_request_id}_#{self.class.base_model.name.underscore}_#{id}"
```

Comment:
@pkomanek Can my_task_id be changed to tracking_id or tracking_label_id? the my_task_id function conflicts with MiqTask. If its used outside of here maybe we could add an alias.

Reply:
Agreed, I mentioned this on the call this morning.  Need to check if it is used elsewhere, but this seems like a good refactoring.